pkg/pb/gracefulexit.proto

```diff
@@ -10,8 +10,41 @@ import "orders.proto";
 
 package gracefulexit;
 
+service NodeGracefulExit {
+  // GetSatellitesList returns a list of satellites that the storagenode has not begun a graceful exit for.
+  rpc GetNonExitingSatellites(GetNonExitingSatellitesRequest) returns (GetNonExitingSatellitesResponse);
+  // StartExit updates one or more satellites in the storagenode's database to be gracefully exiting.
+  rpc StartExit(StartExitRequest) returns (StartExitResponse);
+ 
+}
+
+message GetNonExitingSatellitesRequest{}
+
+// StartExitRequest
+message StartExitRequest {
+  repeated bytes node_ids = 1 [(gogoproto.customtype) = "NodeID", (gogoproto.nullable) = false];
+}
+
+message NonExitingSatellite {
```

Comment:
We should make sure to comment above some of these messages. Specifically `NonExitingSatellite` and `StartExitStatus`